tests/IPBlockTest.php

```diff
@@ -209,4 +211,36 @@ public function testGetSuper()
         } catch (\InvalidArgumentException $e) {
         }
     }
+
+    /**
+     * Tests the generator
+     */
+    public function testGetAddresses()
+    {
```

Comment:
I do not like these 2 tests, they test too many different hings (relies on dynamic computation, humanReadable, one test ipv4 and the other one ipv6, etc.). I would prefer to have a data provider that provides a block and an preset array of expected IPs. Then the test just run the generator and collects all the IP into an array, and assert that the collected array equals the expected array. 16 IPs is still reasonable to write in a data provider, but you can also use smaller blocks. The 2 tests can use the same data provider obviously. What do you think?

Reply:
That seems like a good approach. I'll amend the PR.